Library/brew.rb

```diff
@@ -50,6 +50,15 @@
   EOABORT
 end
 
+if OS.mac? && MacOS.version == "10.11"
+  opoo <<-EOS.undent
+    Although Homebrew has added initial recognition of 10.11, it is unsupported.
+    You may encounter breakage or other failure and there is no guarantee
+    Homebrew can resolve those issues until El Capitan is stable.
+
```

Comment:
Blank line here to make sure the commands aren't buffered against each other, i.e.

```
~> brew info openssl
Warning: Although Homebrew has added initial recognition of 10.11, it is unsupported.
You may encounter breakage or other failure and there is no guarantee
Homebrew can resolve those issues until El Capitan is stable.

openssl: stable 1.0.2a-1 (bottled)
OpenSSL SSL/TLS cryptography library
https://openssl.org
```

Rather than:

```
~> brew info openssl
Warning: Although Homebrew has added initial recognition of 10.11, it is unsupported.
You may encounter breakage or other failure and there is no guarantee
Homebrew can resolve those issues until El Capitan is stable.
openssl: stable 1.0.2a-1 (bottled)
OpenSSL SSL/TLS cryptography library
https://openssl.org
```